spec.html

```diff
@@ -37172,6 +37172,7 @@
       <h1>agent-order</h1>
       <p>For a candidate execution _execution_, _execution_.[[AgentOrder]] is a Relation that satisfies the following conditions.</p>
       <emu-alg>
+        1. Let agent-order be _execution_.[[AgentOrder]].
```

Comment:
This is a good clarification but I don't like the addition to the `let` ecmaspeak grammar here. Can we use \_vars\_?